src/main/java/core/basesyntax/storage/Storage.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.storage;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Integer> fruitStorage = new HashMap<>();
+
+    public static Map<String, Integer> getFruitStorage() {
+        return fruitStorage;
+    }
+
+    public static void addFruit(String fruit, int quantity) {
+        fruitStorage.put(fruit, fruitStorage.getOrDefault(fruit, 0) + quantity);
+    }
+
+    public static void removeFruit(String fruit, int quantity) {
+        fruitStorage.put(fruit, fruitStorage.getOrDefault(fruit, 0) - quantity);
+    }
```

Comment:
The `removeFruit` method should check if the resulting quantity is not negative before updating the storage. According to the checklist, when handling the Purchase operation, you should throw a `RuntimeException` if the balance is negative.